homeassistant/components/tractive/device_tracker.py

```diff
@@ -59,13 +62,16 @@ def __init__(self, user_id, trackable, tracker_details, hw_info, pos_report):
         self._latitude = pos_report["latlong"][0]
         self._longitude = pos_report["latlong"][1]
         self._accuracy = pos_report["pos_uncertainty"]
+        self._source_type = pos_report["sensor_used"]
 
         self._attr_name = f"{self._tracker_id} {trackable['details']['name']}"
         self._attr_unique_id = trackable["_id"]
 
     @property
     def source_type(self):
         """Return the source type, eg gps or router, of the device."""
+        if self._source_type == "PHONE":
+            return SOURCE_TYPE_BLUETOOTH
```

Comment:
Source will be bluetooth if the tracker is close to a phone with the app

Reply:
Should we use `SOURCE_TYPE_ROUTER` if `event["sensor_used"] == "KNOWN_WIFI"`?